workers/unity/Packages/manifest.json

```diff
@@ -5,6 +5,7 @@
     "com.improbable.gdk.gameobjectcreation": "file:com.improbable.gdk.gameobjectcreation",
     "com.improbable.gdk.mobile": "file:com.improbable.gdk.mobile",
     "com.improbable.gdk.playerlifecycle": "file:com.improbable.gdk.playerlifecycle",
+    "com.improbable.gdk.querybasedinterest": "file:com.improbable.gdk.querybasedinterest",
```

Comment:
cc @ElleEss Is this definitely the name we want for this, forever?

Reply:
Bump on this ^

Reply:
@ElleEss another bump